packages/client-utils/src/vstorage.js

```diff
@@ -80,6 +80,10 @@ export const makeVStorage = ({ fetch }, config) => {
   };
 
   /**
+   * Make a vstorage Children or Data query, returning the decoded result along
+   * with corresponding response metadata (cf.
+   * https://docs.cometbft.com/v1.0/spec/abci/abci++_methods#query ).
```

Comment:
according to this documentation the metadata includes much more than blockHeight. Moreover it has no field `blockHeight`, only `height`.

Please either make the code actually return the cited response metadata or correct the doc. E.g.

> returning the decoded result along with the block height from which the data was derived (see `height` in https://docs.cometbft.com/v1.0/spec/abci/abci++_methods#query for more details).